packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -37,8 +38,15 @@ export function isEmailSetUp(): boolean {
 }
 
 export function isUserManagementEnabled(): boolean {
-	// This can be simplified but readability is more important here
+	const license = Container.get(License);
+	const usersLimit = license.getUsersLimit();
+	// If users limit is 1, UM is disabled
+	if (usersLimit === 1) {
```

Comment:
hmm user management should still be enabled if users limit is 1? users should still be able to access login pages and stuff.. just not be able to invite users.. otherwise this would be wrong no? 
```suggestion
	if (usersLimit === 0) {
```